src/routes/+page.svelte

```
@@ -285,6 +306,19 @@
 			destroyFunc.run = unlisten
 		}
 	})
+
+	var toolPaneWidth: number = 20
+	var previousPaneWidth: number = 20
+
+	function handleSnapEase(event: any, pane_index: number, min_size: number, snap_bound: number) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)